src/lib/util.js

```diff
@@ -96,6 +96,9 @@ module.exports = {
     emptyInOutDir: function (dirs) {
         rimraf.sync(`${dirs.in}/*`);
         rimraf.sync(`${dirs.out}/*`);
+        // re-create nested out/errors directory after rimraf-ing its parent dir
+        fse.ensureDirSync(dirs.errors);
```

Comment:
som this will create the errors dir if it doesn't exist yet right?

Reply:
it will recreate it because it gets removed by line 98 above.

Reply:
my only concern with this change is after the last test, an empty errors folder will be created?